3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/eventEnrichmentSpecs.scala

```diff
@@ -65,7 +65,7 @@ class ExtractEventTypeSpec extends Specification with DataTables with Validation
   def e3 =
     "SPEC NAME"          || "INPUT VAL"     | "EXPECTED OUTPUT"                                                                                    |
     "None"               !! None            ! "No collector_tstamp set".fail                                                                       |
-    "Negative timestamp" !! BCTstamp        ! "Collector timestamp -0030-01-01 00:00:00.000 is negative and will fail the Redshift load".fail |
+    "Negative timestamp" !! BCTstamp        ! "Collector timestamp -0030-01-01 01:25:00.000 is negative and will fail the Redshift load".fail |
```

Comment:
@chuwy could you help review I just made the tests pass but I'm not really aware of the implications?